app/locale/en.js

```diff
@@ -425,7 +425,43 @@ module.exports = {
       pricing: 'Pricing',
       help_center: 'Help Center',
       research_efficacy: 'Research - Efficacy',
-      research_impact: 'Research - Impact'
+      research_impact: 'Research - Impact',
+      terms: 'Terms',
+      podcast_simple: 'Podcast',
+      create_free_account: 'Create Free Account',
+      school_district_solutions: 'School & District Solutions',
+      teacher_toolkit_preview: 'Teacher Toolkit Preview',
+      teacher_toolkit: 'Teacher Toolkit',
+      efficacy_studies: 'Efficacy Studies',
+      success_stories: 'Success Stories',
+      hoc: 'Hour of Code',
+      grants_funding_resources: 'Grants & Funding Resources',
+      request_quote_demo: 'Request a Demo or Quote',
+      coco_junior_beta: 'CodeCombat Junior (Beta)',
+      ap_csp: 'AP CSP',
+      ai_league_esports: 'AI League Esports',
+      ai_hackstack_beta: 'AI HackStack (Beta)',
+      ai_hackstack_junior_beta: 'AI HackStack Junior (Beta)',
```

Comment:
We decided to cal this AI Junior instead of AI HackStack Junior.

```suggestion
      ai_junior_beta: 'AI Junior (Beta)',
```